python/cudf/cudf/core/series.py

```diff
@@ -2199,7 +2205,7 @@ def hash_encode(self, stop, use_name=False):
         )
 
         # TODO: Binary op when https://github.com/rapidsai/cudf/pull/892 merged
-        mod_vals = cudautils.modulo(hashed_values.data.to_gpu_array(), stop)
+        mod_vals = cudautils.modulo(hashed_values.data.mem, stop)
```

Comment:
Need to guard against nulls if using numba function, ideally move to new binaryop

Reply:
```suggestion
        mod_vals = hashed_values.binary_operator("mod", stop)
```

Reply:
👍 agreed, could you open an issue so that we have something to track / discuss?